src/chainstate/stacks/auth.rs

```diff
@@ -125,9 +128,8 @@ impl StacksMessageCodec for TransactionAuthField {
     }
 }
 
-impl StacksMessageCodec for MultisigSpendingCondition {
+impl StacksMessageCodec for MultisigTransactionData {
```

Comment:
It's of _paramount importance_ that the existing codec for `MultisigSpendingCondition` behaves the same way before and after this feature activates.  I know it sounds paradoxical, but can you please keep the existing `StacksMessageCodec` in place?  I realize that it will lead to a little bit of code duplication between `MulsitigSpendingCondition` and `MultisigTransactionData`, but I'd rather see that than an accidental consensus bug.  Not changing existing code means not introducing accidental consensus bugs :)